### ISSUE TITLE
KSP Unable to recover vessels due to exception

### ISSUE THREAD

**Issue by Ristellise** [AlphaAsh/Kerbal-Konstructs_DEV#1]
MissingMethodException: Method not found: 'MessageSystem.AddMessage'.
  at KerbalKonstructs.KerbalKonstructs.OnVesselRecovered (.ProtoVessel vessel) [0x00000] in <filename unknown>:0 

  at EventData`1[ProtoVessel].Fire (.ProtoVessel data) [0x00000] in <filename unknown>:0 

  at VesselRetrieval.recoverVessel (.Vessel v) [0x00000] in <filename unknown>:0 

  at VesselRetrieval.recoverVessels () [0x00000] in <filename unknown>:0 

  at VesselRetrieval+.MoveNext () [0x00000] in <filename unknown>:0

**Comment by AlphaAsh** [comment]
Can you reproduce the problem? If so, what are the steps taken.

**Comment by Ristellise** [comment]
step 1 get 1.0.5
step 2 launch any type of craft with  KK installed
step 3 recover it.
step 4 goes back to main menu. Craft icon is still there indicating that it is not recovered.

**Comment by AlphaAsh** [comment]
Followed your steps. Launched a command pod on the launchpad. Recovered it. Got the usual dialogs for recovery. Got the extra message from KK. No pod on the launchpad anymore.
I can't reproduce the problem.

**Comment by Ristellise** [comment]
hmm looks like on my end..

**Comment by TanyaSapien** [comment]
I'm chiming in on this with what I believe to be pertinent data. I'm getting the same error as Joshwoo69 (MessageSystem.AddMessage) however I have clarifications.

I'm attempting to recover a vessel in career mode from a facility other than KSC which was launched from that same facility. When I open the event window to reclaim the vessel, it has a 50/50 chance of displaying 20% recovery value and saying KSC is recovering it, or saying 80% and that Central Lakes Space Operations is recovering it.

Regardless I get stuck in a loop where science rewards and vessel value are being added to my resources, but the vessel itself is not being removed from the world. When this happened, I checked the f12 debug menu and found that error message. I googled it and that led me here.

Interestingly and worth note is that when I booted up the game to reproduce the bug for screenshots, it ran perfectly fine.

Exact steps that created the bug:
Running Kerbal Konstructs with Kerbinside Complete pack (newest build as of 2/2/2016) on KSP 1.0.5
Started a career with all vanilla settings
Purchased CLSC from inside the VAB after building rocket
Launched rocket from CLSC
Performed EVA on surface for extra science
Recovered Jeb
Got error
Attempted recovery from tracking station
Got error
Attempted recovery again of both jebediah and of craft from both tracking center and when controlling as active vessel
Got error
exited game to main menu and reloaded world
Got error
shut down game to post bug report, rebooted game to take screenshots.
Error gone.


I hope this helps in your bug hunting.
